eng/Configurations.props

```diff
@@ -2,6 +2,7 @@
   <!-- Honor the generic RuntimeConfiguration property. -->
   <PropertyGroup>
     <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(ConfigurationGroup)</RuntimeConfiguration>
+    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(Configuration)</RuntimeConfiguration>
```

Comment:
This isn't necessary as `ConfigurationGroup` and `Configuration` are always [passed in together](https://github.com/dotnet/runtime/blob/2cf4433f64cd10070f19de0c091295108a982b7f/eng/build.ps1#L138). In future we will probably remove the former.

Reply:
Not necessarily. If you build the projects directly then `ConfigurationGroup` is not specified. 

Also looking through the logs @jkoritzinsky and I observed places where `$(ConfigurationGroup)` was not set but `$(Configuration)` was. This change was inspired by that observation. It's deep in the restore graph where we encountered this.

Reply:
Concrete example where only `$(Configuration)` is passed to the build 

https://teams.microsoft.com/l/message/19:b27b36ecd10a46398da76b02f0411de7@thread.skype/1580491671080?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=014ca51d-be57-47fa-9628-a15efcc3c376&parentMessageId=1580481868103&teamName=dotnet%2Fruntime%20repo&channelName=Infrastructure&createdTime=1580491671080